.circleci/config.yml

```diff
@@ -18,13 +18,13 @@ executors:
       # Customize the JVM maximum heap limit
       JVM_OPTS: -Xmx3200m
       TERM: dumb
-      CHISEL_REV: origin/master
+      CHISEL_REV: origin/3.4.x
```

Comment:
Is this needed to support MiMa or did this bleed over from another change?

Reply:
Ah yes, sorry those are just used for the testing: https://github.com/freechipsproject/chisel3/blob/03e6c2ba149ac611d1e2329c4502fed0ccea48fe/.circleci/config.yml#L66

The FIRRTL and Treadle variables are needed in case we make source incompatible changes to those projects on master. I don't know why `CHISEL_REV` exists, but I did a diff with what's on `3.3.x` to determine what to change in this file.